sql/core/src/test/resources/sql-tests/results/postgreSQL/float4.sql.out

```diff
@@ -91,291 +91,259 @@ struct<CAST(          -INFINiTY    AS FLOAT):float>
 
 
 -- !query 11
-SELECT float('N A N')
--- !query 11 schema
-struct<CAST(N A N AS FLOAT):float>
--- !query 11 output
-NULL
```

Comment:
I tried it in pgsql and it fails. Let's regenerate the answer files.

Reply:
I have regenerated the answer files and updated them here as well.